Course/Course/Managers/CalendarManager.swift

```diff
@@ -0,0 +1,482 @@
+//
+//  CalendarManager.swift
+//  Course
+//
+//  Created by Shafqat Muneer on 2/20/24.
+//
+
+import Foundation
+import EventKit
+import Theme
+import Core
+import BranchSDK
+
+enum DeepLinkType: String {
```

Comment:
@shafqat-muneer maybe we could re-use `DeepLinkType`  and `DeepLinkKeys` defined in `DeepLink` class to avoid duplication?
Thank you